service/service.go

```diff
@@ -1092,3 +1092,22 @@ func (s *service) validateIsiPath(ctx context.Context, volName string) (string,
 
 	return isiPath, nil
 }
+
+func (s *service) GetIsiPathByName(ctx context.Context, volName string) (string, error) {
+	if s.k8sclient == nil {
+		return "", fmt.Errorf("no k8s clientset")
+	}
+
+	pv, err := s.k8sclient.CoreV1().PersistentVolumes().Get(ctx, volName, v1.GetOptions{})
+	if err != nil {
+		return "", fmt.Errorf("unable to get PersistentVolume: %w", err)
```

Comment:
Does the error include the persistent volume name/ID?

Reply:
Yes @donatwork , Volume name will be included as part of the error.